cms-derived-data/create_cms_derived_datasets.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+# noqa: E501
+
+import ROOT
+import os
+import subprocess
+import yaml
+import json
+import click
+import math
+import re
+
+
+"""
+Create CMS derived datasets.
+"""
+
+
+XROOTD_URI_BASE = "root://eospublic.cern.ch/"
+
+def get_number_events(file_path, data_type):
+    """Return number of events in root file."""
+    myfile = ROOT.TFile.Open(file_path)
+    number_events = 0
+    if data_type == "NanoAODRun1":
+        number_events = myfile.Events.GetEntries()
+    elif data_type == "POET":
+        number_events = myfile.events.GetEntries()
+    return number_events
+
+
+def get_parent_recid(parent_title):
+    """Return parent dataset recid."""
+    cmd = f"cernopendata-client get-metadata --title={parent_title} --output-value recid"
```

Comment:
cernopendata-client will not find the recids for PFNano (unless we can point it to qa content @tibosimko)

Reply:
and it gives me `/bin/sh: line 1: cernopendata-client: command not found` although I have it installed